app/assets/stylesheets/components/_container.sass

```diff
@@ -1,4 +1,4 @@
-$Container-maxWidth: 960px
+$Container-maxWidth: 1440px
```

Comment:
I mean this is the max-width, it will behave the same way it would with 960px. I'm just talking about how long should the content stretch.